client/lib/lib.rs

```diff
@@ -0,0 +1,612 @@
+/// * `deploy` - Deploy arguments container struct holding options for this deploy. See
+///   `DeployStrParams`.
+/// * `session` - Session arguments container struct holding options for this deploy. See
+///   `SessionStrParams`.
+/// * `payment` - Payment arguments container struct holding options for this deploy. See
+///   `PaymentStrParams`.
+pub fn put_deploy(
+    maybe_rpc_id: &str,
+    node_address: &str,
+    verbose: bool,
+    deploy: DeployStrParams<'_>,
+    session: SessionStrParams<'_>,
+    payment: PaymentStrParams<'_>,
+) -> Result<JsonRpc> {
+    let deploy = Deploy::with_payment_and_session(
+        deploy.try_into()?,
+        payment.try_into()?,
+        session.try_into()?,
+    );
+    RpcCall::new(maybe_rpc_id, node_address, verbose)?.put_deploy(deploy)
+}
+
+/// Make a deploy and output to file or stdout.
+///
+/// * `maybe_rpc_id` is used for RPC-ID as required by the JSON-RPC specification, and is returned
+///   by the node in the corresponding response.  It must be either able to be parsed as a `u32` or
+///   empty.  If empty, a random ID will be assigned.
+/// * `node_address` identifies the network address of the target node's HTTP server, e.g.
+///   `"http://127.0.0.1:7777"`.
+/// * When `verbose` is `true`, the request will be printed to `stdout`.
+/// * `deploy` - Deploy arguments container struct holding options for this deploy. See
+///   `DeployStrParams`.
+/// * `session` - Session arguments container struct holding options for this deploy. See
+///   `SessionStrParams`.
+/// * `payment` - Payment arguments container struct holding options for this deploy. See
+///   `PaymentStrParams`.
+#[allow(clippy::too_many_arguments)]
+pub fn make_deploy(
+    maybe_output_path: &str,
+    deploy: DeployStrParams<'_>,
+    session: SessionStrParams<'_>,
+    payment: PaymentStrParams<'_>,
+) -> Result<()> {
+    Deploy::make_deploy(
+        none_if_empty(maybe_output_path),
+        deploy.try_into()?,
+        payment.try_into()?,
+        session.try_into()?,
+    )
+}
+
+/// Transfers an amount between purses.
+///
+/// * `maybe_rpc_id` is used for RPC-ID as required by the JSON-RPC specification, and is returned
+///   by the node in the corresponding response.  It must be either able to be parsed as a `u32` or
+///   empty.  If empty, a random ID will be assigned.
+/// * `node_address` identifies the network address of the target node's HTTP server, e.g.
+///   `"http://127.0.0.1:7777"`.
+/// * When `verbose` is `true`, the request will be printed to `stdout`.
+/// * `amount` - Specifies the amount to be transferred.
+/// * `maybe_source_purse` - Source purse in the sender's account that this amount will be drawn
+///   from, if it is `""`, it will default to the account's main purse.
+/// * `maybe_target_purse` - Target purse in the sender's account that this amount will be
+///   transferred to.
+/// it is incompatible with `maybe_target_account`.
```

Comment:
No need for a new line here, and "it" should be "It".  Same a few lines below too.

Reply:
Thanks, done.